app/modules/node.c

```diff
@@ -643,6 +643,28 @@ static int node_writercr (lua_State *L) {
   lua_pushinteger(L, n);
   return 1;
 }
+
+static int node_int2float(lua_State *L) {
```

Comment:
The changes to this file are not really part of this PR (at least, not by title), but I see no reason they shouldn't also land.

Reply:
I added these as part of testing the new behavior. I can certainly rip them out (I didn't actually intend to commit them). I didn't add documentation for them. I think that they are only relevant for lua53 as well....

What would you like me to do?

Reply:
I think it's fine to leave them in this PR; I was just noting.